nextcord/file.py

```diff
@@ -99,7 +106,7 @@ def __init__(
         *,
         description: Optional[str] = None,
         spoiler: bool = False,
-        force_close: bool = False,
+        force_close: Optional[bool] = None,
```

Comment:
I don't get what changing the default of `force_close` accomplishes?

Reply:
Yeah I think that might be slightly opinionated. The thought process is that if you're using a context manager you want to make sure the file is closed, as leaving it open could result in unexpected behavior.

Reply:
Okay, but changing the default of the `force_close` parameter from `False` to `None` doesn't help. If the default was left alone, then when the context manager is used, the `force_close` attribute is automatically set to `True`, regardless of what its last value was.

Reply:
I did it this way to allow `force_close` to be set to `False` while using the context manager if someone wants to.

Reply:
I see.